mono_repo/lib/src/commands/travis.dart

```diff
@@ -270,20 +270,31 @@ List<String> _calculatePkgEntries(
 Map<String, String> _extractCommands(Map<String, TravisConfig> configs) {
   var commandsToKeys = <String, String>{};
 
-  for (var task in _travisTasks(configs)) {
-    if (commandsToKeys.containsKey(task.command)) {
+  var tasksToConfigure = _travisTasks(configs).toList();
```

Comment:
Minor nit: does this need to be converted to a list, if we're just going to chain functional methods onto it later?

IMO, if we want it to be a list, better to have _travisTasks return a list. Then, we shouldn't need to coerce it here.